tests/unit/customizations/test_scalarparse.py

```diff
@@ -30,10 +35,43 @@ def test_identity(self):
         self.assertEqual(scalarparse.identity(10), 10)
 
     def test_scalar_parsers_set(self):
-        session = mock.Mock()
+        session = Mock()
+        session.get_scoped_config.return_value.get.return_value = 'none'
         scalarparse.add_scalar_parsers(session)
         session.get_component.assert_called_with('response_parser_factory')
         factory = session.get_component.return_value
+        expected = [call(blob_parser=scalarparse.identity),
+                    call(timestamp_parser=scalarparse.identity)]
+        self.assertEqual(factory.set_parser_defaults.mock_calls,
+                         expected)
+
+    def test_choose_none_timestamp_formatter(self):
+        session = Mock()
+        session.get_scoped_config.return_value.get.return_value = 'none'
```

Comment:
Can we do the following to make this more specific and avoid too much mocking magic?
1) Use a botocore session as the spec for the mock
2) Set the get_scoped_config return_value to a dictionary with the timestamp key instead of mocking out the entire get method.